app/utils.py

```diff
@@ -15,6 +15,19 @@
 log = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+# Patches may fail due to cascade delete issue. These are not faults as such so not HoustonExceptions
+# but need to be passed between the parameters file and resources file
+class CascadeDeleteException(Exception):
```

Comment:
Can this exception be documented with an example?  It is not super clear what this exception is responsible for catching, nor an example of why it can be raised.

Reply:
There are examples of use in the individuals and sightings models file in the delete code

Reply:
worse, there is a whole google doc titled _**Cascading object relations in next-gen**_ about this very subject.  :sweat_smile:  i mean if you want some gory details.